tests/rust/build.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import shlex
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--dry-run", action="store_true")
+parser.add_argument("--verbose", action="store_true")
+parser.add_argument("--release", action="store_true")
```

Comment:
Consider adding `help` for each of those, not all the parameters are self-explanatory